docs/7/7.1-configuration-managers.md

```diff
@@ -0,0 +1,81 @@
+# Configuration Managers
+
+### Overview
+
+Configuration Managers (C.M.) are important even in this day of containerization and virtualization. They can be used to set up hosts, create Docker images, create AMIs, and more. Many companies are not yet comfortable with the idea of Docker so this is a nice step in that direction. 
+
+At the end of this section you will: 
+
+- Understand the benefits of configuration management
+- Be able to discuss the main differences and pro/cons of configuration management
+- Compare configuration management with containerization
+- Describe the benefits of immutable infrastructure
+- Be able to explain Infrastructure as Code. 
+
+### Steps
+
+1. Understand Configuration Management
+2. Configuration Management versus Containerization
+3. Immutable Infrastructure
+4. Creating Images with C.M. 
+5. Using Chef with AWS 
+6. Using Ansible with Vagrant
+
```

Comment:
I think the "steps" sections is unnecessary as it is just a toc for a document that is not very long. I think the "end of this section" is more important and having both is confusing.